src/command/player/seasonStatsCompare.js

```diff
@@ -0,0 +1,232 @@
+        seasonObj[currentSeason] = [...Array(playerProfile.length)].fill({});
+        seasonObj[currentSeason][index] = season;
+      }
+    });
+    index += 1;
+  });
+  return seasonObj;
+};
+
+// takes in all the data and returns an array with the player's totals as each entry
+const makeOverall = (playerProfile, seasonStr) => {
+  const overallArr = [];
+  playerProfile.forEach(player => {
+    overallArr.push(...player[`careerTotals${seasonStr}`]);
+  });
+  return overallArr;
+};
+
+// takes in the data for a season and combines the stats into strings to be pushed to table
+/* eslint-disable no-param-reassign */
+const makeRow = seasonData => {
+  const template = {
+    teamAbbreviation: [],
+    playerAge: [],
+    gp: [],
+    min: [],
+    pts: [],
+    fgPct: [],
+    fg3Pct: [],
+    ftPct: [],
+    ast: [],
+    reb: [],
+    stl: [],
+    blk: [],
+    tov: [],
+  };
+  let seasonId;
+  seasonData.forEach(player => {
+    // configure certain data before we append to string
+    // make sure object isn't empty
+    if (Object.keys(player).length !== 0) {
+      player.fg3Pct = (player.fg3Pct * 100).toFixed(1);
+      player.fgPct = (player.fgPct * 100).toFixed(1);
+      player.ftPct = (player.ftPct * 100).toFixed(1);
+      // overall stats dont have team abbreviation, have to check if exists
+      if (player.teamAbbreviation) {
+        const teamMainColor = getMainColor(player.teamAbbreviation);
+        player.teamAbbreviation = chalk`{bold.white.bgHex('${
+          teamMainColor ? teamMainColor.hex : '#000'
+        }') ${player.teamAbbreviation}}`;
+      }
+      if (!template.seasonId) {
+        seasonId = bold(player.seasonId);
+      }
+    }
+    Object.keys(template).forEach(key => {
+      // add data to str or if no data put in a dash
+      template[key].push(player[key] || '-');
+    });
+  });
+
+  Object.keys(template).forEach(key => {
+    // find which player has best stat and color it green
+    const maxInd = findMaxInd(template[key]);
+    template[key][maxInd] = chalk.green(template[key][maxInd]);
+    template[key] = template[key].join('\n');
+  });
+
+  // set seasonId, do this at the end because didnt want key in there when looping
+  template.seasonId = seasonId;
+  return template;
+};
+/* eslint-enable no-param-reassign */
+
+// main function called by index.js
+const seasonStatsCompare = (
+  playerProfile,
+  playerInfo,
+  seasonTtpe = 'Regular Season'
+) => {
+  // these strings are used so we can get the correct data at the beginning and dont have to write same code twice
+  const seasonStr = seasonTtpe.replace(/\s/g, '');
+
+  // compile the data into a format to push to the table
+  const seasonObj = makeSeasonObj(playerProfile, seasonStr);
+  const overallArr = makeOverall(playerProfile, seasonStr);
+  const nameStr = makeNameStr(playerInfo, seasonTtpe);
+
+  // make sure we are adding rows in the correct order
+  const seasonDates = Object.keys(seasonObj).sort((a, b) => {
```

Comment:
This may be replaced by [R.keys](http://ramdajs.com/docs/#keys) and [R.sort](http://ramdajs.com/docs/#sort)